src/components/plots/plot.ts

```diff
@@ -36,7 +41,10 @@ module Plottable {
     private _extentsProvider: Scales.ExtentsProvider<any>;
 
     protected _animate: boolean = false;
-    private _animators: Animators.PlotAnimatorMap = {};
+    private _animators: Animators.PlotAnimatorMap = {
+      "main": new Animators.Base(),
+      "reset": new Animators.Null()
```

Comment:
It thinks that `_animators` is a duplicate variable. I ended up taking the first line of your suggestion and initializing the latter two in the constructor, where they probably should belong anyways.